benchmarks/csharp/csharp_benchmark.csproj

```diff
@@ -15,6 +15,14 @@
     <TargetFrameworks>net6.0;net8.0</TargetFrameworks>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
+    <LangVersion>preview</LangVersion>

```

Comment:
Otherwise we will have to update language version every-time we will need to allow new syntax (and there is no good reason to disable new syntax - in dotnet old syntax remains availible)

Reply:
Yeah, and actually for .net 6 default version of language not the same as for .net 8, so it would lead to usage old syntax net8 wise (also no preference in having old syntax)